packages/solo/src/start.js

```diff
@@ -107,12 +107,14 @@ async function buildSwingset(
     );
 
     // eslint-disable-next-line import/no-dynamic-require,global-require
-    return require(pluginFile);
+    return import(pluginFile);
```

Comment:
Does the `eslint-disable-next-line` mode need to be changed? Or does dynamic `import` fall into the same category as `no-dynamic-require`?